Lib/turtledemo/__main__.py

```diff
@@ -96,8 +96,10 @@
 
 import turtle
 
+
```

Comment:
Remove this line:

```suggestion
```

Reply:
Correct.  Double space only recommended for function or class definition.

Reply:
Well, it's mostly because we shouldn't make stylistic changes to existing code until pep 8 is enforced.